README.md

```diff
@@ -25,7 +25,7 @@ Note, too, that since the SDK does not validate parameters on method calls, app
 - Controllable Mock Firebolt server (NodeJS websocket + HTTP server)
 - Control mock method responses via control plane RESTful API, CLI, web admin UI (soon), browser extension (soon)
 - 100% OpenRPC-driven; no SDK-specific details within the implementation
-- Supports Firebolt SDKs: **core** (always), **manage** (optional), **discovery** (optional)
+- Supports Firebolt SDKs: **core** (always), **discovery** (optional)
```

Comment:
I still think we should say we support the manage SDK, since we do. It's just that the OpenRPC for it is included in the one unified OpenRPC file.

If you don't want to mention the 'manage' stuff, then I think instead of 'SDK' you have to say 'OpenRPC' or something like that, not SDK.

Reply:
Maybe we can change manage to "(always)" since it's loaded with core now.

Reply:
Addressed